inspector.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Flutter Widget Inspector
+
+permalink: /inspector/
+---
+
+The Flutter Widget Inspector is powerful tool for visualizing and exploring
+Flutter Widget trees.
+
+* TOC Placeholder
+{:toc}
+
+## Flutter Widget Inspector
+
+The Flutter framework uses Widgets as [the core building
+block](https://flutter.io/widgets-intro/) for anything from controls (text,
+buttons, toggles, etc.) to layout (centering, padding, rows, columns, etc.). The
+Flutter Widget Inspector is powerful tool for visualizing and exploring Flutter
+these Widget trees. It can be helpful when:
+
+* Undestanding existing layouts
+* Determining which source code produces a particular widget
+* Diagnosing layout issues
+
+For a demonstration of the Widget Inspector, please see the recent
+[DartConf talk](https://www.youtube.com/watch?v=JIcmJNT9DNI).
+
+## Get Started with the Inspector
+
+The Widget Inspector is currently available in [the Flutter
+plugin](https://flutter.io/ide-setup/) for Android Studio, or IntelliJ IDEA.
```

Comment:
Screenshot for which button to click to activate inspector?

Reply:
In the new M22 build launching this week it auto-activated whenever you run an app!